playbook/app/pb_kits/playbook/pb_avatar_action_button/avatar_action_button.rb

```diff
@@ -2,13 +2,11 @@
 
 module Playbook
   module PbAvatarActionButton
-    class AvatarActionButton
-      include Playbook::Props
-
+    class AvatarActionButton < Playbook::KitBase
       prop :action, type: Playbook::Props::String,
                     default: "add"
       prop :image_url, type: Playbook::Props::String
-      prop :link_url, type: Playbook::Props::String
+      prop :link_url, type: Playbook::Props::String, default: "#"
```

Comment:
```suggestion
      prop :link_url, type: Playbook::Props::String
```

This change seems unrelated to the PR.

Reply:
@thestephenmarshall the link being `nil` makes the `link_to` in the view generate a URL for the current page, making it always link somewhere regardless of the explicit desire of the kit user. The problem was exposed in this change because the view renderer is trying to generate a URL using the Rails application routes, while the route where this kit is rendered is covered by the Engine, causing an error.

Reply:
Ah ok then if we were receiving an error pls ignore my comment.